controlplane/controllers/remediation.go

```diff
@@ -373,8 +373,8 @@ func (r *CK8sControlPlaneReconciler) checkRetryLimits(log logr.Logger, machineTo
 	return remediationInProgressData, true, nil
 }
 
-// max calculates the maximum duration.
-func max(x, y time.Duration) time.Duration {
+// maxDuration calculates the maximum duration.
```

Comment:
```suggestion
// maxDuration returns the longer of two time.Duration values.
```
I think the original godoc is also quite odd, because there's no calculation involved.